scripts/deploy-live-comms.sh

```diff
@@ -0,0 +1,33 @@
+
+#!/bin/sh
+
+# source deploy-util.sh
```

Comment:
Should this be removed? I don't see this file in your branch.

Reply:
Removed it, I'll remove it from the other deploy scripts too. I copied it as a template. Maybe worth thinking about abstracting the deployment behavior after we've now done 3 different deployments, and we'll be talking about even more of that as we consider putting nginx infront of django? Part of the spike probably to think about it.